src/fw/drivers/imu/lsm6dso/lsm6dso.c

```diff
@@ -25,22 +25,25 @@
 #include "lsm6dso_reg.h"
 #include "lsm6dso.h"
 
-// Forward declaration of private functions defined below public functions
+#define MIN(a, b) ((a) < (b) ? (a) : (b))
```

Comment:
already in `src/libutil/includes/util/math.h`

Reply:
also please avoid combo commits like 07d4f866c66cbefb29c512c5f9612d45aa4a82a7 where many things are changed, making review difficult.